hudi-spark-datasource/hudi-spark/src/test/scala/org/apache/spark/sql/hudi/TestCreateTable.scala

```diff
@@ -62,6 +62,53 @@ class TestCreateTable extends TestHoodieSqlBase {
     )(table.schema.fields)
   }
 
+  test("Test Create Hoodie Table With Options") {
+    val tableName = generateTableName
+    spark.sql(
+      s"""
+         | create table $tableName (
+         |  id int,
+         |  name string,
+         |  price double,
+         |  ts long,
+         |  dt string
+         | ) using hudi
+         | partitioned by (dt)
+         | options (
```

Comment:
we would change here to tblproperties and rename the test description?

Reply:
Here is a UT for test the compatibility to use `options`. We should show a standard example for users in official document.